cli/pkg/providers/aws/aws.go

```diff
@@ -51,6 +52,18 @@ func (p *provider) GetOverrides() tmpl.Variables {
 	}
 }
 
+func (p *provider) CheckEnvironment() (bool, string) {
```

Comment:
nitpicks:
- It's more idiomatic in go to return an `err` than a string. Also may as well return the error directly and let the function caller add the "Please check..." string.
- `IsEnvironmentValid` seems a bit better since this method returns a boolean